tests/unit/contrib/test_dpucahx8l.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 Xilinx Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for testing the DPUCAHX8L build functionality"""
+
+import os
+import unittest
+import numpy as np
+
+from pyxir import partition
+from pyxir.graph.layer.xlayer import XLayer, ConvData
+from pyxir.graph.partitioning.xgraph_partitioner import XGraphPartitioner
+from pyxir.graph.xgraph_factory import XGraphFactory
+from pyxir.target_registry import TargetRegistry
+from pyxir.runtime.rt_manager import RtManager
+
+from .compilation_infra import (
+    xcompiler_conv2d_pool2d_nhwc_oihw_test,
+    xcompiler_scale_conv2d_nhwc_oihw_test,
+    xcompiler_resnetv1_block_test,
+)
+
+try:
+    import tensorflow as tf
+
+    skip_tf = False
+except ModuleNotFoundError:
+    skip_tf = True
+
+
+class TestDPUCAHX8H(unittest.TestCase):
+
+    xgraph_partitioner = XGraphPartitioner()
+    xgraph_factory = XGraphFactory()
+    target_registry = TargetRegistry()
+    rt_manager = RtManager()
+
+    @classmethod
+    def setUpClass(cls):
+        import pyxir.contrib.target.DPUCAHX8L
+
+    @classmethod
+    def tearDownClass(cls):
+        # Unregister dpu for other tests
+        TestDPUCAHX8H.target_registry.unregister_target("DPUCAHX8L-u50")
+        TestDPUCAHX8H.target_registry.unregister_target("DPUCAHX8L-u280")
+
+    def test_compile_conv2d_pool2d(self):
+        xcompiler_conv2d_pool2d_nhwc_oihw_test(
```

Comment:
You will have to add the targets here, default is DPUCAHX8H

Reply:
Added targets